harness/determined/exec/prep_container.py

```diff
@@ -24,12 +25,11 @@ def trial_prep(sess: api.Session, info: det.ClusterInfo) -> None:
     model_def_resp = None
     try:
         model_def_resp = bindings.get_GetModelDef(sess, experimentId=trial_info.experiment_id)
-    except bindings.APIHttpError as e:
+    except api.errors.MasterNotFoundException as e:
```

Comment:
I dont think this change will work. If you look at the underlying code, SSLError should still also be meaningful, possibly other that aren't caught explicitly https://github.com/determined-ai/determined/blob/e1971c814cf7522133ab1442654ea2e2e4a7d8af/harness/determined/common/api/request.py#L142-L147

Reply:
I'll put it back to Exception which is what it was before the DET-8003 change.

Reply:
that one doesn't make sense to me either, tbh (cc @rb-determined-ai maybe it is his code)